indico/modules/events/timetable/templates/pdf/_cover_page.html

```diff
@@ -0,0 +1,24 @@
+{% from 'events/display/conference/_util.html' import format_event_date %}
+
+{% macro render_cover_page(event) %}
+    <div class="pagebreak cover">
+        <div class="cover-wrapper">
+            <div>
+                {% if event.logo %}
+                    <img alt="Event logo" src="event://logo" class="logo">
+                {% endif %}
+                <h1>{{ event.title }}</h1>
+                <h3>{{ format_event_date(event) }}</h3>
+                <h1>{% trans %}Programme{% endtrans %}</h1>
```

Comment:
Source strings are not british

```suggestion
                <h1>{% trans %}Program{% endtrans %}</h1>
```